### PR TITLE
[Android] Allowing timeout to be configured through PlayerRuntimeConfig

### PR DESCRIPTION
**Pull request by A1shK**
### Change Type (required)
While trying to debug, the app would crash due to [this](https://github.com/player-ui/player/blob/main/jvm/j2v8/src/main/kotlin/com/intuit/playerui/j2v8/extensions/Lock.kt#L13) timeout. This PR moves the timeout to the `PlayerRuntimeConfig`.

- [x] `patch`
- [ ] `minor`
- [ ] `major`

### Does your PR have any documentation updates?
- [ ] Updated docs
- [ ] No Update needed
- [ ] Unable to update docs

NOTE: Happy to add some documentation around this. However, I cant find a suitable place in the docs site to add this. Please let me know whats the best place to add some documentation to.

### CODE REVIEW

**Comment by intuit-svc** [comment]
Canary builds can only be requested by collaborators. Please tag a maintainer to request a canary build. ❌

File: docs/site/pages/getting-started.mdx [hunk 1]
```diff
@@ -115,8 +115,19 @@ var body: some View {
...
+val player = AndroidPlayer(
+    listOf(
+        ReferenceAssetsPlugin(),
+        // Any other plugins
+    )
+)
 ```
+Apart from providing a list of plugins while setting up `AndroidPlayer`, you can also provide a `config` object that has the following options:
+ - `debuggable` - Flag to indicate if this is a debug build
+ - `coroutineExceptionHandler` - Custom coroutine exception handler
```
Comment (brocollie08) [inline docs/site/pages/getting-started.mdx:127]: this exception handler should be handling all uncaught exceptions while using the runtime coroutine scope

File: docs/site/pages/getting-started.mdx [hunk 2]
```diff
@@ -115,8 +115,19 @@ var body: some View {
...
-val player = AndroidPlayer(ReferenceAssetsPlugin())
+val player = AndroidPlayer(
+    listOf(
+        ReferenceAssetsPlugin(),
+        // Any other plugins
+    )
+)
 ```
+Apart from providing a list of plugins while setting up `AndroidPlayer`, you can also provide a `config` object that has the following options:
+ - `debuggable` - Flag to indicate if this is a debug build
```
[the remark this answers is not in the document]
Comment (A1shK) [reply docs/site/pages/getting-started.mdx:126]: Thanks! This is a good explanation...time to copy-pasta 🙇

**Review by brocollie08 — APPROVED** [review]

**Review by sugarmanz — APPROVED** [review]

**Pull request merged**